arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1586,12 +1699,12 @@
       "x-ms-azure-resource": true,
       "description": "The Resource definition for other than namespace."
     },
-    "NamespaceListResult": {
+    "SBNamespaceListResult": {
```

Comment:
Why is this name changing?  I don't see this in PR#944.  Same question applies to other type name changes.  This will generate different type names in the various SDKs which can be a breaking change.